enhancements/insights/pulling-sca-certs-from-ocm.md

```diff
@@ -84,12 +91,19 @@ available in the `pull-secret` (in the `openshift-config-managed` namespace).
 
 The Insights Operator must provide a cluster ID as an identifier of the cluster.
 
-### Data in API
+### SCA certs in API
 
-The SCA certificate is available via the `etc-pki-entitlement` secret in the `openshift-config-managed` namespace.
+The SCA certificate is available via the `etc-pki-entitlement` secret in the `openshift-config-managed` namespace. The secret will be available for use in other namespaces by creating a cluster-scoped `Share` resource. Cluster admin creates a `clusterrolebinding` to allow a service account access to the `Share` resource.
```

Comment:
link to the csi-shared-resource driver page, especially since this isn't something that just exists on clusters today:

https://github.com/openshift/csi-driver-shared-resource

Reply:
Linked at https://github.com/openshift/enhancements/pull/683/files#diff-93dffd2aa59c973a916306675f62907275cff29ff9a16ba77aaf98cfa830249fR100